src/nvim/highlight_group.c

```diff
@@ -231,90 +223,67 @@ static const char *highlight_init_both[] = {
   "default link DiagnosticUnnecessary      Comment",
 
   // Treesitter standard groups
-  "default link @variable            NONE",  // don't highlight to reduce visual overload
```

Comment:
I think that all top-level groups should have explicit definitions; this is important so 1. users see them and know to style them and 2. for fallbacks (not all of which are listed) to work.

Reply:
Apart from `@variable`, I think that only applies to `@character`.